duply/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM debian:buster
```

Comment:
The version that comes with debian stretch `1.11.3` has a bug with `file://` backups, which we use in the test. This is fixed in `2.0.2` and included in debian buster.
See https://duply.net/wiki/index.php/Duply-Changelog

Reply:
:+1: